csi/server/plugin/opensds/controller.go

```diff
@@ -178,24 +187,28 @@ func (p *Plugin) CreateVolume(
 	} else {
 		createVolume, err := Client.CreateVolume(volumebody)
 		if err != nil {
-			isExist, isCompatible, findV, findErr := FindVolume(volumebody)
-			if findErr != nil {
-				return nil, findErr
-			}
-
-			if !(isExist && isCompatible) {
-				glog.Error("failed to CreateVolume", err)
-				return nil, err
-			}
-
-			v = findV
-			glog.V(5).Infof("Although the return failed, it was actually successful. volume = %v", findV)
+			glog.Error("failed to CreateVolume", err)
```

Comment:
Please make this log more clear like for example, `glog.Error("failed to CreateVolume: ", err)` or `glog.Errorf("failed to CreateVolume: %v\n", err)`.

Reply:
Already modified, please review again.